src/parameterizations/vertical/MOM_geothermal.F90

```diff
@@ -35,6 +35,8 @@ module MOM_geothermal
   type(time_type), pointer :: Time => NULL() !< A pointer to the ocean model's clock.
   type(diag_ctrl), pointer :: diag => NULL() !< A structure that is used to
                                              !! regulate the timing of diagnostic output.
+  integer :: id_internal_heat_tend_3d = -1 ! ID for 3D diagnostic of internal heat
```

Comment:
Variables in types and functions need to be documented, so Doxygen need this Comment needs to start with `!<`.

Also would be good to align the `!<` with the comment above.